pkgs/shells/yash/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "yash";
-  version = "2.54";
+  version = "2.55";
 
   src = fetchurl {
-    url = "https://osdn.net/dl/yash/yash-${version}.tar.xz";
-    hash = "sha256-RKCsHM98Os7PvqAn2MDJMPE6goBlvjGAVc4RMBU5GDk=";
+    url = "https://github.com/magicant/yash/releases/download/${version}/yash-${version}.tar.xz";
+    hash = "sha256-l82AnV4hazxK+uQjefG9T1CCt8FtUeKC1gpQFPvJ4fY=";
   };
```

Comment:
Can this become a `fetchFromGitHub` since we get the source from github.com?

Reply:
Tried switching to it, but for some reason it can't find `xgettext`:

```
xgettext -d yash -D .. -cTRANSLATORS: -f - -F --no-wrap \
        --package-name=yash --package-version=2.55 \
        --msgid-bugs-address=http://osdn.jp/projects/yash/forums/ \
        --copyright-holder='Magicant' -kgt -kNgt -kngt:1,2 --flag=sb_vprintf:1:c-format --flag=sb_printf:1:c-format --flag=malloc_vprintf:1:c-format --flag=malloc_printf:1:c-format --flag=xerror:1:c-format --flag=serror:1:c-format --flag=le_compdebug:1:c-format
/nix/store/9b4skch97sv02rwmczbxp1pwg382hv84-bash-5.2-p15/bin/bash: line 2: xgettext: command not found
```

Reply:
ah, got it - was missing asciidoc and gettext as a nativebuildinput